jenkins/Jenkinsfile

```
@@ -55,7 +79,52 @@ pipeline {
             '''
           }
         }
+        stage('NDTensorscuTENSORExt julia-1.11') {
```

```suggestion
        stage('NDTensorscuTENSORExt julia-1') {
```
since if I understand the workflow it will use whatever is the latest release, which at some point will not be 1.11.

---

updated